host-interaction/bootloader/manipulate-boot-configuration.yml

```diff
@@ -4,6 +4,8 @@ rule:
     namespace: host-interaction/bootloader
     author: william.ballenthin@mandiant.com
     scope: function
+    att&ck:
+      - Defense Evasion::Impair Defenses [T1562]
```

Comment:
```suggestion
```

this rule we *had* mapped but decided the rule doesn't specifically imply an impact. there's another rule or two that are more related to disabling boot or other security features. so, i think we should keep this rule unmapped.